src/uct/ib/dc/accel/dc_mlx5.c

```diff
@@ -353,6 +354,49 @@ ucs_status_t uct_dc_mlx5_ep_atomic_cswap32(uct_ep_h tl_ep, uint32_t compare, uin
                                  htonl(compare), htonl(swap), comp);
 }
 
+ucs_status_t uct_dc_mlx5_ep_atomic_post(uct_ep_h ep, unsigned opcode, uint64_t value,
+                                        size_t size, uint64_t remote_addr, uct_rkey_t rkey)
+{
+    switch (opcode) {
+    case UCT_ATOMIC_POST_OP_ADD:
+        switch (size) {
+        case sizeof(uint32_t):
+            return uct_dc_mlx5_ep_atomic_add32(ep, value, remote_addr, rkey);
+        case sizeof(uint64_t):
+            return uct_dc_mlx5_ep_atomic_add64(ep, value, remote_addr, rkey);
+        default:
+            ucs_assertv(0, "invalid ATOMIC_ADD size: %zu", size);
```

Comment:
need to add capability bits to indicate support for atomic operations

Reply:
right, but it seems there is no bit-space in iface caps

Reply:
that's right, probably need another bit field just for atomics